### DISCUSSION TITLE
Pixi name and color

### DISCUSSION THREAD

**Discussion by a1xon** [pixijs/pixijs#7313] [Ideas]
Hey I have a really strange problem with "pixi".
I built a business application with it and it works pretty well actually. The most problematic thing is when I say that it's based on pixi they think the name is somewhat childish and the color looks girly. So they don't take the project serious.
Yep - that's stupid. I know that. But is there an opportunity to have a fork with a stronger name "rockSolid2DEngine"? and different main color? I'm pretty sure there a devs out there facing the same problem.

**Comment by bigtimebuddy** [comment]
You can turn off the hello banner in your console in the utils if you don't want to advertise. Also, the name is "PixiJS" not "pixi". Uppercasing is more professional.

A core audience for the project has been and continues to be games and rich visual experiences, not enterprise applications. We feel like the branding fit to that audience well.

Lastly, the idea that pink is "girly" and undermines the seriousness of our project is not only incorrect but sexist. We think our branding differentiates us. If you want a use a gray-blue mark and boring name, there are countless other open source projects out there that fit that.

Appreciate the feedback, but not interested in changing.

**Reply by a1xon** [reply]
Don't get me wrong please. I love this framework and I like the color & name.

**Comment by GoodBoyDigital** [comment]
Heya! If the success or failure of a project hinges on our color scheme being to 'girly' there are definitely larger issues afoot with said project! :P

**Reply by a1xon** [reply]
true...

**Comment by mreinstein** [comment]
RockSolidMetalMonsterTruckLightingTorqueBeefSteakRenderer2D

**Reply by a1xon** [reply]
"Military Grade"

**Comment by dylanpomeroy** [comment]
Devs just add this to the page: 
![image](https://user-images.githubusercontent.com/12665583/111689392-5908e780-8802-11eb-95cc-5a7693bc9ccf.png)

**Reply by GoodBoyDigital** [reply]
https://www.youtube.com/watch?v=ighZE_VyHIc

**Comment by ivanpopelyshev** [comment]
Good idea, maybe I can publish fork for production and name it like that :)
